ledger-api/grpc-definitions/com/daml/ledger/api/v1/transaction_filter.proto

```diff
@@ -47,13 +47,13 @@ message InclusiveFilters {
   // SHOULD NOT contain duplicates.
   // All ``template_ids`` need to be valid: corresponding templates should be defined in one of
   // the available packages at the time of the query.
-  // Deprecated in favor of the `template_filters`. If the `template_filters` field is set,
-  // the `template_ids` field is ignored.
+  // Deprecated in favor of the ``template_filters``. If the ``template_filters`` field is set,
+  // the ``template_ids`` field is ignored.
   // Optional
   repeated Identifier template_ids = 1 [deprecated = true];
 
   // Include an ``InterfaceView`` for every ``InterfaceFilter`` matching a contract.
-  // The ``InterfaceFilter``s MUST use unique ``interface_id``s.
```

Comment:
This doesn't build anymore due to [this change](https://github.com/digital-asset/daml/pull/17606/files#diff-bd749e6ba10cc644e517ea2592f798738ab24063cf525393bcc53247073e4bdbL55-L56). Specifically, the removal of the next line changes the resulting RST line from:
```rst
     - Include an ``InterfaceView`` for every ``InterfaceFilter`` matching a contract. The ``InterfaceFilter``s MUST use unique ``interface_id``s. All ``interface_id`` needs to be valid: corresponding interface should be defined in one of the available packages at the time of the query. Optional
```
to
```rst
     - Include an ``InterfaceView`` for every ``InterfaceFilter`` matching a contract. The ``InterfaceFilter``s MUST use unique ``interface_id``s. Optional
```
At first glance that seems OK, but, as we can see in the following screenshot of the current live website, that doesn't get parsed quite as one would hope:
<img width="410" alt="Screenshot 2023-11-01 at 17 12 54" src="https://github.com/digital-asset/daml/assets/45561385/94cc7bac-fc50-4abe-b7ab-4d4c152ab1da">
The `literal` string beginning with
```rst
The ``InterfaceFilter
```
was actually ending with
```rst
interface_id``
```
which is no longer there.

Reply:
The syntax highlighting here on GitHub actually shows the unexpected parsing.